settings.gradle.kts

```diff
@@ -8,3 +8,4 @@ pluginManagement {
 
 include(":java-11")
 include(":java-14")
```

Comment:
Please either keep java 14 up to date, or remove it (which I think is fine since it was added at the time as the "latest" java, we shouldn't be using it)